suite-native/app/src/App.tsx

```diff
@@ -51,7 +52,7 @@ export const App = () => {
 
     return (
         <GestureHandlerRootView style={{ flex: 1 }}>
-            <NavigationContainer>
+            <NavigationContainer onReady={() => SplashScreen.hide()}>
```

Comment:
please use handler for this, like `const handleNavigationReady = () => { ... }`

Reply:
https://github.com/trezor/trezor-suite/pull/5839/commits/3c0080e66d6eae35d3fac435e5a8287760d02b5a